src/main/java/world/bentobox/bentobox/util/IslandInfo.java

```diff
@@ -56,9 +56,7 @@ public void showAdminInfo(User user, Addon addon) {
                     TextVariables.UUID, owner.toString());
 
             // Fixes #getLastPlayed() returning 0 when it is the owner's first connection.
-            long lastPlayed = (Bukkit.getOfflinePlayer(owner).getLastSeen() != 0)
-                    ? Bukkit.getOfflinePlayer(owner).getLastSeen()
-                    : Bukkit.getOfflinePlayer(owner).getLastSeen();
+            long lastPlayed = Bukkit.getOfflinePlayer(owner).getLastSeen();
```

Comment:
This looks like a work around based on the comments to a bug where the system reports 0 when it's the player's first time, but that seems correct...